ecolex/legislation.py

```diff
@@ -165,21 +154,20 @@ def harvest_file(upfile):
             else:
                 for lang_field in LANGUAGE_FIELDS:
                     legislation[lang_field].add(lang)
-                logger.error('Language not found %s' % (lang))
+                logger.error(f'Language not found {lang}')
```

Comment:
Please also add in the dictionary Slovak, Catalán, Sinhala and others showing up in the error log